libs.ts

```diff
@@ -846,7 +884,11 @@ export const libs: Lib[] = [
 		userApps: 'Yes',
 		polls: 'Yes',
 		forwarding: 'Yes',
-		appEmoji: 'Yes'
+		appEmoji: 'Yes',
+		componentsV2: {
+			text: 'Has a PR',
+			url: 'https://github.com/discord-jda/JDA/pull/2806'
+		}
```

Comment:
looks like this was replaced by https://github.com/discord-jda/JDA/pull/2809

Reply:
done, forgot about that